ocfweb/docs/docs/staff/backend/libvirt.md

```diff
@@ -0,0 +1,78 @@
+[[!meta title="KVM/Libvirt"]]
+
+Most of the OCF's hosts are virtual machines. Our virtual machines are all
+powered by [QEMU][qemu]/[KVM][kvm] and managed by [libvirt][libvirt] on the
+hypervisors.
+
+VM disks are stored as LVM volumes on the hypervisors, typically under
+`/dev/vg/<vm-name>`.
+
+[qemu]: https://www.qemu.org/
+[kvm]: https://www.linux-kvm.org/
+[libvirt]: https://libvirt.org/
+
+
+## Administration
+
+### How do I view which VMs are on a hypervisor?
+
+SSH into the hypervisor, then run `sudo virsh list --all`:
+
+    kpengboy@hal:~$ sudo virsh list --all
+     Id    Name                           State
+    ----------------------------------------------------
+     1     fallingrocks                   running
+     3     limniceruption                 running
+     4     spriggs                        running
+     5     dev-flood                      running
+     11    dev-whiteout                   running
+     15    dev-death                      running
+     18    dev-tsunami                    running
+     -     dev-anthrax                    shut off
+     -     dev-pestilence                 shut off
+     -     dev-werewolves                 shut off
+     -     zombies                        shut off
+
+### How do I turn on a VM?
+
+On the hypervisor, run `sudo virsh start <vm-name>`.
+
+### How do I turn off a VM?
+
+You can SSH into the VM and run the `poweroff` command, or you can run
```

Comment:
I think using `shutdown -h now` here is better, especially since it's one of the commands that `ocfroot` users can run without entering a password and because it runs through any shutdown scripts, which `poweroff` apparently doesn't do.

Reply:
On the second point, `poweroff` is an alias for `shutdown` except with the `-f` option.